src/hotspot/cpu/aarch64/globals_aarch64.hpp

```diff
@@ -88,6 +88,8 @@ define_pd_global(intx, InlineSmallCode,          1000);
           "Use Neon for CRC32 computation")                             \
   product(bool, UseCRC32, false,                                        \
           "Use CRC32 instructions for CRC32 computation")               \
+  product(bool, UseCryptoPmull, false,                                  \
```

Comment:
```suggestion
  product(bool, UsePmullForCRC32, false,                                  \
```

Reply:
> Changed to UseCryptoPmullForCRC32 as there is another pmull version.

Can you give me a reference for that? I can only find one definition of pmull.

Reply:
There will be two pmull implementations
1) `-XX:+UseCryptoPmullForCRC32` to use this 64x64 pmull implementation
2) `-XX:-UseCryptoPmullForCRC32 -XX:-UseCRC32 -XX:+UseNeon` to use the [8x8 pmull implementation](https://github.com/openjdk/jdk/blob/1ef9f6507ba45419f0fa896915eec064762c5153/src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp#L3753).